.pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-job-template.yaml

```diff
@@ -61,10 +61,22 @@ stages:
         parameters:
           sub: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
           clusterName: ${{ parameters.clusterName }}-$(commitID)
-          os: ${{ parameters.os }}
+          os: linux
```

Comment:
I see in master branch, we only pass linux as an OS
I think we should not hard code it here to linux, but leave it as parameter
https://github.com/Azure/azure-container-networking/blob/e874736dba7b9d0e2ffbbf7cbcf2a1241555f070/.pipelines/pipeline.yaml#L455